components/NumbersList/NumberItem.js

```diff
@@ -16,7 +16,7 @@ export default function NumberItem({
   const { getDate } = useTimestamp()
 
   const calledDate = called && showDate && calledAt && getDate(calledAt)
-
+ 
```

Comment:
Remove this empty line (and the line in 65)

Reply:
sorry, it is okay to have one line separating `calledDate` from `saveNotes` (line 18) and another line between `expandedArea` and `date` (line 65)